tests/test_pools_gauges.py

```diff
@@ -4,8 +4,17 @@
 
 EXAMPLE_PREFERENTIAL_GAUGES = {
```

Comment:
How did you pick these examples?  Did you find edge cases/problem children?  I think if we asked people to think about it we could come up with a few.